### DISCUSSION TITLE
box out of range in rotate function for Yolo training

### DISCUSSION THREAD

**Discussion by kronee0516** [albumentations-team/albumentations#1269] [Q&A] [answered]
How to prevent the box get out of the image when using rotate function?
any setup to do before processing the image?
or i can only do it by verifying the processed box ?

**Comment by Dipet** [comment, marked as answer]
Try [`SafeRotate`](https://albumentations.ai/docs/api_reference/augmentations/geometric/rotate/#albumentations.augmentations.geometric.rotate.SafeRotate)
